index.js

```diff
@@ -1,3 +1,4 @@
 'use strict';
 
+// The real stuff is all in /lib
```

Comment:
This change can be reverted see commit "Core library change to prove that a full CI test cycle is run"

Reply:
Ah your right. Good spot.